modules/ROOT/pages/dataweave-versioning-behavior.adoc

```diff
@@ -0,0 +1,17 @@
+= Versioning Behavior in DataWeave
+
+In each version of DataWeave, changes are introduced that modify the way the language behaves. When existing scripts depend on previous behavior, system properties are introduced to toggle the change so that backwards compatibilty is not broken in the current version. These system properties that control behavior in DataWeave are called *Compatibility Flags*. Each flag defined a deletion version, after which it will no longer exists, and the behavior that it controlled will default to the new one.
+
+Specifying DataWeave's *language level* indicates to the runtime which Compatibility Flags are available. As previously stated, when upgrading to a new version there are flags that will be deleted. In order to facilitate updatating DataWeave, setting the language level of a previous version makes it so that the flags available on that version are visible, mantaining backwards compatibility. Language level defaults to the running DataWeave version and has to be at most the same as the runtime version.
```

Comment:
```suggestion
Specifying a DataWeave language level indicates to the DataWeave runtime what compatibility flags are available. To upgrade and maintain backward compatibility, you can set the language level of a previous version so that the flags available on that version remain visible even if they expire in the DataWeave version to which you upgrade. The language level defaults to the running DataWeave version and has to be at most the same as the runtime version.
```

Reply:
@svacas What does "and has to be at most the same as the runtime version" mean? Do you mean that the DW version cannot exceed the DataWeave runtime version that you are running? Or what? Thx!

Reply:
It means the DataWeave runtime version running. In a mule app it is the DW that is packaged in the application.